content/developers/internals/data-model/usage-example.en.md

```diff
@@ -0,0 +1,273 @@
+
+Here are the track sections that we then need:
+
+![Track sections schema](../svg_schemas/small_infra_rails.drawio.en.svg)
+
+{{% alert title="Important" color="warning"%}}
+They are represented as arrows in our schema to stress the fact that they have a **start** and an **end**. This is important because the objects related to the track section are positioned using an offset, and the offset is a distance from the **start** of the track section.
+
+Therefore, if you want to place an object at the beginning of the track section, you would set its offset to 0. To place it at the end of the track section, its offset should be the `length` of the track section.
+{{% /alert %}}
+
+For the objects to be complete, there are different attributes to set:
+
+* `length`: the length of the track section in meters.
+* `geo`: the coordinates in real life (geo is for geographic), in the [GeoJSON](https://en.wikipedia.org/wiki/GeoJSON) format.
+* `sch`: the coordinates in the schematic view (sch for schematic), also in [GeoJSON](https://en.wikipedia.org/wiki/GeoJSON) format.
+* cosmetic attributes: `line_name`, `track_name`, `track_number` which are used to indicate the name and labels that were given to the tracks / lines in real life.
+
+For all track sections in our infrastructure, the `geo` and `sch` attributes are identical, and very much resemble the given schema.
+
+For most track sections, their `length` is proportional to what can be seen in the schema, but for readability, we made *TA6*, *TA7*, *TD0* and *TD1* not correspond to their length (which are 10km and 25km).
+
+### Track Section Links
+
+For the moment we only created track sections, they have length and coordinates but they are not linked to each other (we do not check which coordinates overlap, thank you very much). We simply indicate the link between two track sections by creating an object, linking one extremity to the other.
+
+If two track sections are not connected by a `TrackSectionLink`, a train in an *OSRD* simulation cannot go from the one to the other.
+
+In our example, creating the links is straight-forward: since we chose to have long track sections, we only have to link them at switches. Happily, in our generation scripts, like in [small_infra.py](https://github.com/DGEXSolutions/osrd/blob/dev/core/examples/generated/scripts/small_infra.py), when creating switches, the corresponding `TrackSectionLink`s are automatically created.
+
+Let's delve a bit on the switches.
+
+### Switches
+
+<!-- TODO: what it is -->
+The switches are the parts needed to link more than two different track sections together.
+
+A switch manages the different links that are associated to it (always at least two). In general, it translates into the following rule: when one link is used, the others cannot be used at the same time. This is because if two links were used by trains at the same time, it would be an accident.
```

Comment:
> (always at least two)

We can define a SwitchType with only one port (and it makes sense for cross switches.

Reply:
Same question as in [comment below](https://github.com/DGEXSolutions/osrd-website/pull/28#discussion_r899891960), I think I don't really understand the use of groups